packages/react-scripts/config/webpack.config.js

```diff
@@ -484,6 +489,13 @@ module.exports = function (webpackEnv) {
                   isEnvDevelopment &&
                     shouldUseReactRefresh &&
                     require.resolve('react-refresh/babel'),
+                  enableIstanbulPlugin && [
+                    require.resolve('babel-plugin-istanbul'),
+                      {
+                        // do not instrument code coverage for tests / storybook files
+                        exclude: ["**/*.test.*", "**/*.stories.*"],
```

Comment:
This is provided by the plugin: https://github.com/istanbuljs/babel-plugin-istanbul#ignoring-files